batch7crud-roll5/src/main/java/com/lftechnology/batch7crud/util/MD5Encryption.java

```diff
@@ -0,0 +1,35 @@
+package com.lftechnology.batch7crud.util;
+
+import java.math.BigInteger;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.lftechnology.batch7crud.dao.EmployeeDAOImpl;
+import com.lftechnology.batch7crud.exception.EncryptionException;
+
+public class MD5Encryption {
+    private static final Logger LOGGER = Logger.getLogger(EmployeeDAOImpl.class.getName());
+
+    private MD5Encryption() {
+
+    }
+
+    public static String getMD5(String input) throws NoSuchAlgorithmException, EncryptionException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: java.security.NoSuchAlgorithmException, !unknownSymbol! [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)